app/views/ViewUtils.scala

```diff
@@ -16,16 +16,51 @@
 
 package views
 
-import play.api.data.Form
+import play.api.data.{Field, Form, FormError}
 import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.html.components.{RadioItem, Text}
+import viewmodels.RadioOption
 
 object ViewUtils {
 
   def errorPrefix(form: Form[_])(implicit messages: Messages): String = {
-    if (form.hasErrors || form.hasGlobalErrors) s"${messages("error.browser.title.prefix")}" else ""
+    if (form.hasErrors || form.hasGlobalErrors) s"${messages("error.browser.title.prefix")} " else ""
   }
 
   def breadcrumbTitle(title: String)(implicit messages: Messages): String = {
-    s"$title - ${messages("entities.trustDetails")} - ${messages("site.service_name")} - GOV.UK"
+    s"$title - ${messages("entities.otherIndividual")} - ${messages("site.service_name")} - GOV.UK"
```

Comment:
Presume this needs changing back?

Reply:
Yea. Also I think it should just be `service.name` and remove `site.service_name` from the messages keys & only use service.name to avoid confusion of having both?